Added front matter to tutorial and fixed apostrophes

nigelcharman opened the pull request:
(required UTF-8 encoding)

Tutorial should now be available at concordion.github.io/concordion-website-2.0/tutorial. Might need to configure Jekyll sites to get the /tutorial link to point to this?

ShaKaRee commented:
see individual commits within concordion-net project history:
https://github.com/concordion/concordion-net/commits/gh-pages

ShaKaRee commented:
@nigelcharman would you review these please, and merge if you're happy with them

nigelcharman commented:
I started making line comments on these commits, then changed over to making the comments on the individual commits. Mostly grammatical and spelling comments. thanks

ShaKaRee commented:
@nigelcharman thank you for all your great comments.
I have updated the documentation and added some comments.
Could you merge, if you are happy with the changes?
I am keen to provide the information about the new release on the website soon.

nigelcharman commented:
Great. I've merged it now, thanks.

On 11/03/15 20:41, ShaKaRee wrote:
>
> @nigelcharman <https://github.com/nigelcharman> thank you for all your 
> great comments.
> I have updated the documentation and added some comments.
> Could you merge, if you are happy with the changes?
> I am keen to provide the information about the new release on the 
> website soon.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/concordion/concordion-website/pull/3#issuecomment-78217658>.
>

nigelcharman committed:
Added front matter to tutorial and fixed apostrophes (required UTF-8 encoding)
